File: CS/XtraTreeListHotTrack/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraTreeList;
using DevExpress.XtraTreeList.Nodes;

namespace XtraTreeListHotTrack
{
    public partial class Form1 : Form
    {
        private TreeListNode hotTrackNode = null;

        public Form1()
        {
            InitializeComponent();
        }

        private TreeListNode HotTrackNode
        {
            get { return hotTrackNode; }
            set
            {
                if (hotTrackNode != value)
                {
                    TreeListNode prevHotTrackNode = hotTrackNode;
                    hotTrackNode = value;
                    if (treeList1.ActiveEditor != null)
                        treeList1.PostEditor();
                    treeList1.RefreshNode(prevHotTrackNode);
                    treeList1.RefreshNode(hotTrackNode);
                }
            }
        }

        private void treeList1_NodeCellStyle(object sender, GetCustomNodeCellStyleEventArgs e)
        {
            if (e.Node == HotTrackNode)
                e.Appearance.BackColor = Color.LightBlue;
        }

        private void treeList1_MouseMove(object sender, MouseEventArgs e)
        {
            TreeList treelist = sender as DevExpress.XtraTreeList.TreeList;
            TreeListHitInfo info = treelist.CalcHitInfo(new Point(e.X, e.Y));
            HotTrackNode = info.HitInfoType == HitInfoType.Cell ? info.Node : null;
        }

        private void treeList1_MouseLeave(object sender, EventArgs e)
        { HotTrackNode = null; }
    }
}

File: VB/XtraTreeListHotTrack/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraTreeList
Imports DevExpress.XtraTreeList.Nodes

Namespace XtraTreeListHotTrack
	Partial Public Class Form1
		Inherits Form
		Private hotTrackNode_Renamed As TreeListNode = Nothing

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Property HotTrackNode() As TreeListNode
			Get
				Return hotTrackNode_Renamed
			End Get
			Set(ByVal value As TreeListNode)
				If hotTrackNode_Renamed IsNot value Then
					Dim prevHotTrackNode As TreeListNode = hotTrackNode_Renamed
					hotTrackNode_Renamed = value
					If treeList1.ActiveEditor IsNot Nothing Then
						treeList1.PostEditor()
					End If
					treeList1.RefreshNode(prevHotTrackNode)
					treeList1.RefreshNode(hotTrackNode_Renamed)
				End If
			End Set
		End Property

		Private Sub treeList1_NodeCellStyle(ByVal sender As Object, ByVal e As GetCustomNodeCellStyleEventArgs) Handles treeList1.NodeCellStyle
			If e.Node Is HotTrackNode Then
				e.Appearance.BackColor = Color.LightBlue
			End If
		End Sub

		Private Sub treeList1_MouseMove(ByVal sender As Object, ByVal e As MouseEventArgs) Handles treeList1.MouseMove
			Dim treelist As TreeList = TryCast(sender, DevExpress.XtraTreeList.TreeList)
			Dim info As TreeListHitInfo = treelist.CalcHitInfo(New Point(e.X, e.Y))
			HotTrackNode = If(info.HitInfoType = HitInfoType.Cell, info.Node, Nothing)
		End Sub

		Private Sub treeList1_MouseLeave(ByVal sender As Object, ByVal e As EventArgs) Handles treeList1.MouseLeave
			HotTrackNode = Nothing
		End Sub
	End Class
End Namespace
